CONTRIBUTING.md

```diff
@@ -45,15 +45,16 @@ deck.gl development is governed by the vis.gl Technical Steering Committee (TSC)
 
 - [Ib Green](https://github.com/ibgreen)
 - [Xiaoji Chen](https://github.com/Pessimistress) - chairperson
-- [Tarek Sherif](https://github.com/tsherif)
-- [Ravi Akkenapally](https://github.com/1chandu)
+- [Alberto Asuero](https://github.com/alasarr)
 
 
 ### Maintainers
 
 - [Andrew Duberstein](https://github.com/ajduberstein) - pydeck, json
 - [Xintong Xia](https://github.com/xintongxia) - layers
 - [Georgios Karnas](https://github.com/georgios-uber) - glTF
+- [Kyle Barron](https://github.com/kylebarron) - tiles
+- [Chris Gervang](https://github.com/chrisgervang) - terrain
 - [Dario D'Amico](https://github.com/damix911) - ArcGIS
 - [Jesús Botella](https://github.com/jesusbotella) - MVT
```

Comment:
@alasarr Feel free to nominate someone else (MVTLayer/carto module)

Reply:
Done!